spec/features/user_signs_in_spec.rb

```diff
@@ -10,6 +10,7 @@
 
     visit new_user_session_path
 
+    fill_in 'User name', with: user.user_name
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.